sceptre/nlpsandbox/config/common/nlpsandboxvpc.yaml

```diff
@@ -0,0 +1,12 @@
+template_path: remote/vpc-mini.yaml
+stack_name: nlpsandboxvpc
```

Comment:
you may need to delete the existing stack before deploying this.  I've found that AWS isn't super great about updating many VPC resources at once particularly a CIDR change.